<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialoglisteressources</class>
 <widget class="QDialog" name="Dialoglisteressources">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>861</width>
     <height>631</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="lineEdit_rechercher">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_tri">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableview">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>141</y>
      <width>781</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AjtR">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModR">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SupR">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonRetourner">
    <property name="geometry">
     <rect>
      <x>602</x>
      <y>570</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Retourner</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
